### DISCUSSION TITLE
Got Error When `import '@khmyznikov/pwa-install';`

### DISCUSSION THREAD

**Discussion by jihad-gdt** [khmyznikov/pwa-install#83] [Q&A] [answered]
Hi @khmyznikov, when I call the import I got this error. can you help me?

I just import it by following the documentation on my component like this:

I use react.js with this version `"react": "^16.13.1",`

`import '@khmyznikov/pwa-install';`

<img width="365" alt="image" src="https://github.com/user-attachments/assets/8f70d21c-1b72-4181-9538-f8308e4858cd">

thanks!

**Comment by khmyznikov** [comment, marked as answer]
Check the react example link in readme
